static/src/js/containers/AdminRetrievalsContainer/__tests__/AdminRetrievalsContainer.test.js

```diff
@@ -52,6 +52,16 @@ describe('mapDispatchToProps', () => {
     expect(spy).toBeCalledTimes(1)
   })
 
+  test('onFetchAdminRetrievals calls actions.fetchAdminRetrievals with userId', () => {
+    const dispatch = jest.fn()
+    const spy = jest.spyOn(actions, 'fetchAdminRetrievals')
+
+    mapDispatchToProps(dispatch).onFetchAdminRetrievals('testuser')
+
+    expect(spy).toBeCalledTimes(1)
```

Comment:
`.toBeCalled*` is deprecated, and should be rewritten to `.toHaveBeenCalled*`. Update all of those calls in this file